cmake/std/pr_config/pullrequest.ini

```diff
@@ -0,0 +1,366 @@
+use SEMS-ENV:
+use ATDM-ENV:
+module-load sems-gcc: 4.9.3
+use SEMS-DEFAULT:
+use ATDM-DEFAULT:
+module-remove sems-hdf5:
+module-remove sems-netcdf:
+module-remove sems-openmpi:
+module-remove sems-parmetis:
+module-remove sems-scotch:
+module-load sems-hdf5: 1.10.6/base
+module-load sems-netcdf: 4.7.3/base
+module-load sems-metis: 5.1.0/base
+setenv OMP_NUM_THREADS: 1
+
+
+
+
+[Trilinos_pullrequest_gcc_7.2.0]
+use SEMS-ENV:
+use ATDM-ENV:
+module-load sems-gcc: 7.2.0
+use SEMS-DEFAULT:
+use ATDM-DEFAULT:
+setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 4.9.3
+
+
+
+[Trilinos_pullrequest_gcc_7.2.0_debug]
+use Trilinos_pullrequest_gcc_7.2.0:
+
+
+
+[Trilinos_pullrequest_gcc_7.2.0_serial]
+use SEMS-ENV:
+module-load sems-git: 2.10.1
+module-load sems-gcc: 7.2.0
+module-load sems-python: 2.7.9
+module-load sems-boost: 1.63.0/base
+module-load sems-zlib: 1.2.8/base
+module-load sems-hdf5: 1.10.6/base
+module-load sems-netcdf: 4.7.3/base
+module-load sems-metis: 5.1.0/base
+module-load sems-superlu: 4.3/base
+module-load sems-cmake: 3.10.3
+use ATDM-ENV:
+module-load atdm-ninja_fortran: 1.7.2
+setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 4.9.3
+setenv OMP_NUM_THREADS: 2
+
+
+
+[Trilinos_pullrequest_gcc_8.3.0]
+use SEMS-ENV:
+use ATDM-ENV:
+module-load sems-gcc: 8.3.0
+module-load sems-ninja_fortran: 1.10.0
+use SEMS-DEFAULT:
+module-remove sems-cmake:
+module-remove sems-boost:
+use ATDM-DEFAULT:
+module-load sems-boost: 1.66.0/base
+module-load sems-cmake: 3.17.1
+setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 4.9.3
+
+
+
+[Trilinos_pullrequest_intel_17.0.1]
+use SEMS-ENV:
+use ATDM-ENV:
+module-load sems-gcc: 5.3.0
+module-load sems-intel: 17.0.1
+module-load sems-mpich: 3.2
+use SEMS-DEFAULT:
+module-remove sems-openmpi:
+use ATDM-DEFAULT:
+setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 4.9.3
+
+
+
+[Trilinos_pullrequest_clang_7.0.1]
+use SEMS-ENV:
+use ATDM-ENV:
+module-load sems-gcc: 5.3.0
+module-load sems-clang: 7.0.1
+use SEMS-DEFAULT:
+use ATDM-DEFAULT:
+module-load sems-cmake: 3.12.2
+setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 5.3.0
+
+
+
+[Trilinos_pullrequest_clang_9.0.0]
+use SEMS-ENV:
+use ATDM-ENV:
+module-load sems-gcc: 5.3.0
+module-load sems-clang: 9.0.0
+use SEMS-DEFAULT:
+use ATDM-DEFAULT:
+module-remove sems-cmake:
+module-load sems-cmake: 3.12.2
+setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 5.3.0
+
+
+
+[Trilinos_pullrequest_clang_10.0.0]
+use SEMS-ENV:
+module-load sems-git: 2.10.1
+module-load sems-gcc: 5.3.0
+module-load sems-clang: 10.0.0
+module-load sems-openmpi: 1.10.1
+module-load sems-python: 2.7.9
+module-load sems-boost: 1.69.0/base
+module-load sems-zlib: 1.2.8/base
+module-load sems-hdf5: 1.10.6/parallel
+module-load sems-netcdf: 4.7.3/parallel
+module-load sems-parmetis: 4.0.3/parallel
+module-load sems-scotch: 6.0.3/nopthread_64bit_parallel
+module-load sems-superlu: 4.3/base
+module-load sems-cmake: 3.17.1
+module-load sems-ninja_fortran: 1.10.0
+
+setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 5.3.0
+setenv OMP_NUM_THREADS: 2
```

Comment:
Should this be using `SEMS-DEFAULT` and then modifying it?

Reply:
It could. But I didn't :p